website/files/models/base.py

```diff
@@ -12,7 +12,6 @@
 from modularodm.exceptions import NoResultsFound
 from dateutil.parser import parse as parse_date
 
-from api.base.utils import absolute_reverse
```

Comment:
@chrisseto has a PR where he changes several absolute_reverse calls too. First in wins.